src/sys/dune

```diff
@@ -0,0 +1,22 @@
+;; Test of the Sys library
+
+;; this prevents the tests from running on a default build
+
+(alias
+ (name default)
+ (package multicoretests)
+ (deps stm_tests.exe lin_tests.exe lin_tests_dsl.exe))
```

Comment:
You only need `(deps stm_tests.exe)` here as there aren't test files `lin_tests.ml` or `lin_tests_dsl.ml` in this sub-directory